### PR TITLE
Stimulus controller structure and stimulus-rails gem

### PR DESCRIPTION
**Pull request by martinemde**
This loads, but otherwise ignores stimulus. I'll use this as a base to convert each of the javascript features to stimulus.

### CODE REVIEW

File: config/importmap.rb [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
 
 pin "application", preload: true
+pin "jquery" # @3.7.1
+pin "@rails/ujs", to: "@rails--ujs.js" # @7.1.3
+# stimulus.min.js is a compiled asset from stimulus-rails gem
+pin "@hotwired/stimulus", to: "stimulus.min.js"
+# stimulus-loading.js is a compiled asset only available from stimulus-rails gem
+pin "@hotwired/stimulus-loading", to: "stimulus-loading.js"
+pin_all_from "app/javascript/controllers", under: "controllers"
+pin "clipboard", preload: false # @2.0.11
```
Comment (segiddins) [inline config/importmap.rb:11]: Why the change to preload false?

**Review by segiddins — APPROVED** [review]

File: config/importmap.rb (hunk 1 above)
Comment (martinemde) [reply config/importmap.rb:11]: These are only used on specific pages. Seemed aggressive to preload them. I think this change snuck in with my overall changes though and wasn't really intended.

Do you think I should revert this?

Comment (segiddins) [reply config/importmap.rb:11]: It's fine so long as it's intentional and you've tested that it doesn't break the clipboard functionality

Comment (martinemde) [reply config/importmap.rb]: I'll revert for now. I want to cleanup the loading but I'll do it in a separate PR.

File: config/importmap.rb [hunk 2]
```diff
@@ -1,13 +1,18 @@
 # Pin npm packages by running ./bin/importmap
 
-pin "application", preload: true
```
Comment (martinemde) [inline config/importmap.rb:3]: Note: `preload: true` is the default.
